### ISSUE TITLE
cant fit name into space for banned names

### ISSUE THREAD

**Issue by SilverBullet1234** [triplea-game/triplea#9104]
in the "toolbox"

the spot where you enter names that should be banned does not fit long names.

**Comment by zlefin** [comment]
I checked with silver, and the exact limit it currently accepts is 20 chars; I think I found the spot that's the source
at
/triplea/game-core/src/main/java/games/strategy/engine/lobby/moderator/toolbox/tabs/banned/names/BannedUsernamesTab.java

in the method   buildAddUsernameBanPanel
has maxlength set at 20.  

I'm not sure why it has a maxlength set at all or if there's any problems raising the maxlength, or just not having a maxlength.  All the other code I see nearby about the ban names support strings of any length.

I can't find Swing api info on the jtextfieldbuilder or why it would have a limit on it.

I don't see any obvious problems with just raising the maxlength, but who knows what else is in the code somewhere.
